data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1275,5 +1275,83 @@
     "autolearn": true,
     "qualities": [ { "id": "CHISEL", "level": 2 }, { "id": "HAMMER", "level": 2 } ],
     "components": [ [ [ "material_rhodonite", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chem_phenol",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": "50 m",
+    "batch_time_factors": [ 85, 3 ],
+    "book_learn": [ [ "adv_chemistry", 3 ], [ "textbook_chemistry", 3 ], [ "recipe_labchem", 3 ], [ "reference_cooking", 6 ] ],
+    "charges": 100,
+    "//": "hydrolysis of acetylsalicylic to salicylic acid followed by thermal decomposition to phenol",
+    "qualities": [ { "id": "CHEM", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 120, "LIST" ] ] ],
+    "components": [ [ [ "aspirin", 200 ] ], [ [ "chem_sulphuric_acid", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chem_phenol",
+    "id_suffix": "by_pyrolysis",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 7,
+    "time": "30 m",
+    "book_learn": [ [ "adv_chemistry", 5 ] ],
+    "charges": 100,
+    "//": "pyrolysis of coal, 600-800C, add/substitute coal chemistry books if they appear, other phenols and cresols can be byproducts",
+    "qualities": [ { "id": "CHEM", "level": 3 }, { "id": "HAMMER", "level": 3 } ],
+    "tools": [ [ [ "forging_standard", 10, "LIST" ] ], [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "tongs", -1 ] ] ],
+    "components": [ [ [ "coal_lump", 50 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chem_dnp",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 6,
+    "time": "2 h 30 m",
+    "batch_time_factors": [ 90, 4 ],
+    "charges": 250,
+    "book_learn": [ [ "recipe_labchem", 5 ], [ "reference_cooking", 8 ], [ "reference_explosives", 3 ] ],
+    "qualities": [ { "id": "CHEM", "level": 3 } ],
+    "//": "one of the safest explosive producing nitrations, if somewhat slow",
+    "tools": [ [ [ "surface_heat", 50, "LIST" ] ] ],
+    "components": [
+      [ [ "chem_phenol", 200 ] ],
```

Comment:
Missing water, KNO2. Saltpeter is KNO3, not nitrite.

Also missing a HUGE amount of water, or ice water, or plenty of time, to take on all the heat your source wants to make to not explode your glassware when you dilute the nitric acid. The book also wants you to cool the mixture as it's forming the intermediate. 



This also produces nitrogen oxide as a byproduct. These should either be: Scrubbed with a scrubber setup that's not in game, consumed by consuming filter mask charges, or dealt with safely by doing the work in a fumehood. Oh, or dealt with by the survivor inhaling them and getting serious lung injuries. 

The cooling isn't really something we have in game yet as part of recipes... 


The stoichometry appears to be a little off:   47 g phenol : 43 g KNO2: 21 g NaOH: 224 mL/(purity nitric acid used) of nitric acid. I'm not currently in a place to be able to balance the reaction and give the ratio of DNP produce, but I trust you'll be able to do so for your PR? I can assist later if not.


And, as erk says: this really needs to consume some analysis charges. Your source even mentioned the MP to check.

Reply:
> Missing water, KNO2. Saltpeter is KNO3, not nitrite.
> 
Thermal decomposition with iron as catalyst, abstracted - to be revised if kno2 becomes useful in more reactions

> Also missing a HUGE amount of water, or ice water, or plenty of time, to take on all the heat your source wants to make to not explode your glassware when you dilute the nitric acid. The book also wants you to cool the mixture as it's forming the intermediate.

Perhaps bit more water, definitely not huge amounts. Dilution can be done in controlled manner with stirring to reduce heat generation.

> This also produces nitrogen oxide as a byproduct. These should either be: Scrubbed with a scrubber setup that's not in game, consumed by consuming filter mask charges, or dealt with safely by doing the work in a fumehood. Oh, or dealt with by the survivor inhaling them and getting serious lung injuries.

Can be simply done outside or by the open window, most any reaction involving HNO3 does so.

> The cooling isn't really something we have in game yet as part of recipes...

Hence that requirement is omitted.
 
> The stoichometry appears to be a little off: 47 g phenol : 43 g KNO2: 21 g NaOH: 224 mL/(purity nitric acid used) of nitric acid. I'm not currently in a place to be able to balance the reaction and give the ratio of DNP produce, but I trust you'll be able to do so for your PR? I can assist later if not.

Better to have nice round numbers than perfect accuracy, as long as its close enough. Though I might be convinced for going stichiometrically accurate once all reagents are converted to ml/grams and have concentrations defined.

> And, as erk says: this really needs to consume some analysis charges. Your source even mentioned the MP to check.

No, you need a scale and a thermometer (like for almost ANY reaction). And for .5C precision test one does not need to go to F-J aparatus - a test tube, some oil and thermometer will do.